sdk/common/src/main/java/io/opentelemetry/sdk/internal/DefaultExceptionAttributeResolver.java

```diff
@@ -15,10 +16,19 @@
  */
 public final class DefaultExceptionAttributeResolver implements ExceptionAttributeResolver {
 
+  private static final String ENABLE_JVM_STACKTRACE_PROPERTY =
+      "otel.experimental.sdk.jvm_stacktrace";
+
   private static final DefaultExceptionAttributeResolver INSTANCE =
-      new DefaultExceptionAttributeResolver();
+      new DefaultExceptionAttributeResolver(
+          Boolean.parseBoolean(ConfigUtil.getString(ENABLE_JVM_STACKTRACE_PROPERTY, "false")));
```

Comment:
Added new config to allow toggling between the naive JVM rendering logic and the new limits aware stacktrace rendering logic. 

Given the solid testing in `StackTraceRendererTest`, I think we should default to the new limits aware rendering logic.